api/v1/verticadb_types.go

```diff
@@ -341,6 +346,8 @@ const (
 	CommunalInitPolicyCreate = "Create"
 	// The database in the communal path will be initialized in the VerticaDB
 	// through a revive_db.  The communal path must have a preexisting database.
+	// This option could also be used to initialize the database by restoring
+	// from a database archive if restorePoint field is properly specified.
 	CommunalInitPolicyRevive = "Revive"
```

Comment:
I thought we were going to add a new initPolicy option `Restore`.

Reply:
As discussed, we would like to proceed with a simpler design where we reuse `Revive` for the purpose of restore at this moment. We can revisit this design choice later if necessary.